src/activity_actor.cpp

```diff
@@ -374,6 +375,41 @@ std::unique_ptr<activity_actor> migration_cancel_activity_actor::deserialize( Js
     return migration_cancel_activity_actor().clone();
 }
 
+void open_gate_activity_actor::start( player_activity &act, Character & )
+{
+    act.moves_total = moves;
+    act.moves_left = moves;
+    act.placement = placement;
```

Comment:
We don't really need to set this as it never gets used.
```suggestion
```

Reply:
Ok, sounds good.  I included that line as line 248 of gates.cpp had
`p.activity.placement = pos;` 
and I wasn't sure exactly where `player_activity::placement` was used. If we are removing that line, should we also just get rid of the `open_gate_activity_actor::placement` member variable?

Reply:
well, I think you'll need it to call `gates::open_gate()` in the finish function :D

Reply:
Whoops, of course! And that explains where `player_activity::placement` was used. Thanks!